gemrb/core/Scriptable/Actor.cpp

```diff
@@ -3657,7 +3657,7 @@ void Actor::PlaySelectionSound(bool force)
 	if (!found) {
 		ResRef sound;
 		GetSoundFromFile(sound, Verbal::Select);
-		core->GetAudioDrv()->Play(sound, SFX_CHAN_MONSTER, Pos);
+		core->GetAudioDrv()->Play(sound, SFX_CHAN_MONSTER, Pos, GEM_SND_SPATIAL);
```

Comment:
I'm suspicious whether most of these Actor ones should be spatial. Armor clangs are fine as that, but if your pc is getting mauled off-screen, the player should know about it. For selection is likely doesn't matter, for regular combat sounds no idea, should be tested.

Reply:
Good point, trying out and maybe make it conditional.